include/z64effect.h

```diff
@@ -82,13 +82,15 @@ typedef struct {
 } EffectBlureElement; // size = 0x18
 
 typedef struct {
-    /* 0x000 */ UNK_TYPE1 pad0[0x184];
-    /* 0x184 */ Color_RGBA8 unk184;
-    /* 0x188 */ Color_RGBA8 unk188;
-    /* 0x18C */ Color_RGBA8 unk18C;
-    /* 0x190 */ Color_RGBA8 unk190;
-    /* 0x194 */ UNK_TYPE1 pad194[0xC];
-} EffBlureInit1; // size = 0x1A0
+    /* 0x000 */ char unk_00[0x184];
+    /* 0x184 */ u8 p1StartColor[4];
+    /* 0x188 */ u8 p2StartColor[4];
+    /* 0x18C */ u8 p1EndColor[4];
+    /* 0x190 */ u8 p2EndColor[4];
```

Comment:
So these have to be arrays instead of `Color_RGBA8`?

Reply:
These had to be arrays in OoT because they were set with for loops in some actors (note that only the "init" structs have them as arrays), so I left it this way for now, but I guess we'll have to see if it remains needed in MM